src/install.sh

```diff
@@ -1242,7 +1242,8 @@ EOF
         "Small"|"Medium"|"Large")
             save_response SIZE_PROFILE $opt
             echo "Using profile: '${SIZE_PROFILE}'"
-            source sizing_profiles/${SIZE_PROFILE}.env
+            file=`echo ${SIZE_PROFILE} | tr [:upper:] [:lower:]`
```

Comment:
do we prefer using `$()` synatx vs backticks?

Reply:
I'm not a bash expert, not sure.  😛